aries-site/src/pages/learn/how-to-easily-add-search-and-filter-to-datatable.mdx

```diff
@@ -0,0 +1,365 @@
+import { Box, Image, Notification } from 'grommet';
+
+<Notification
+  status="info"
+  width="large"
+  margin={{ bottom: 'medium' }}
+  message='This guide uses a DataTable that is configured to fetch data from a REST API. If you’d like to learn more about how this configuration works, check out the guide on "How to fetch data from an API and render it in a DataTable."'
+  actions={[
+    {
+      label: 'Guide TBD',
+      href: '#',
+    },
+  ]}
+/>
+
+Let’s get started!
+
+Here's a [CodeSandbox template](https://codesandbox.io/s/hpe-design-system-template-lrdtpe?file=/index.js) with the required dependencies already installed.
+
+Copy and paste this paginated DataTable code into your CodeSandbox. You can replace all of the code.
+
+```
+import React, { useEffect, useState } from "react";
+import { createRoot } from "react-dom/client";
+
+import { Box, DataTable, Grommet, Pagination, Text } from "grommet";
+import { hpe } from "grommet-theme-hpe";
+
+const fetchLaunches = async (view) => {
```

Comment:
This feels like a lot of code that is secondary to the main concepts we are trying to get across. I wonder if we should either trim down the example to not need this, hard code for now, or pick a different REST API endpoint that requires less code.

Reply:
Agree, I was feeling like the API related functions were starting to dominate a bit as well -- I'll explore the options you proposed.

Reply:
Updated the guide to simplify a bit, let me know what you both think. I think it feels more "Data-centric" now.